source/includes/connect/connection-targets.kt

```diff
@@ -0,0 +1,58 @@
+package org.example
+import com.mongodb.ConnectionString
+import com.mongodb.MongoClientSettings
+import com.mongodb.ServerAddress
+import com.mongodb.ServerApi
+import com.mongodb.ServerApiVersion
+import com.mongodb.kotlin.client.MongoClient
+
+fun main() {
+    // start-connect
+    val connectionString = ConnectionString("<connection string URI>")
+
+    // Defines Stable API version
+    val serverApi = ServerApi.builder()
+        .version(ServerApiVersion.V1)
+        .build()
+
+    // Uses MongoClientSettings to apply connection string and specify the Stable API version
+    val settings = MongoClientSettings.builder()
+        .applyConnectionString(connectionString)
+        .serverApi(serverApi)
+        .build()
+
+    val mongoClient = MongoClient.create(settings)
+    // end-connect
+
+    // start-connect-local
+    val connectionString = ConnectionString("mongodb://localhost:27017")
+
+    val settings = MongoClientSettings.builder()
+        .applyConnectionString(connectionString)
+        .build()
+
+    val mongoClient = MongoClient.create(settings)
+    // end-connect-local
+
+    // start-connect-rs-connection-string
+    val connectionString = ConnectionString("mongodb://host1:27017,host2:27017,host3:27017/")
+    val mongoClient = MongoClient.create(connectionString)
```

Comment:
As above don't have to wrap it in a connection string - the factory helper will do that for the user.